### ISSUE TITLE
"single window mode" problem

### ISSUE THREAD

**Issue by unuruu5760** [yuzu-emu/yuzu#7446]
I use two or more monitors. Changing the YUZU "single window mode" to "full screen" by pressing F11 always only works on the first monitor.
I want to change this to "full screen" on my 2nd or 3rd monitor.
please...

**Comment by thethiny** [comment]
Just drag the window to the screen u want and press F11 🤔 works for me.

**Comment by unuruu5760** [comment]
> Just drag the window to the screen u want and press F11 🤔 works for me.

yeah , I hit F11 after moving the window to my 2nd monitor but it goes full screen on 1st monitor...

**Comment by unuruu5760** [comment]
> Just drag the window to the screen u want and press F11 🤔 works for me.

Run yuzu on the first monitor, drag only single window mode(game), move it to the second monitor, and press f11. 

I want to put the yuzu main window on the 1st monitor and change the single window mode (game) to fullscreen on the 2nd monitor.

**Comment by thethiny** [comment]
Still can't reproduce your problem. However go into Windows Settings and change your primary display, that solves it.

**Comment by unuruu5760** [comment]
> 여전히 문제를 재현할 수 없습니다. 그러나 Windows 설정으로 이동하여 기본 디스플레이를 변경하면 문제가 해결됩니다.

I am using a virtual monitor. It seems that Yuzu did not recognize this as a monitor.
https://github.com/roshkins/IddSampleDriver

**Comment by thethiny** [comment]
> > 여전히 문제를 재현할 수 없습니다. 그러나 Windows 설정으로 이동하여 기본 디스플레이를 변경하면 문제가 해결됩니다.
> 
> I am using a virtual monitor. It seems that Yuzu did not recognize this as a monitor. https://github.com/roshkins/IddSampleDriver

Yuzu will not enumerate windows not enumerated by the OS itself. Make sure to go into settings and "detect" and make sure the monitor is there.

**Comment by unuruu5760** [comment]
> > > 일이 일어날 수 있습니다. Windows 설정으로 이동하기 시작하시면 됩니다.
> > 
> > 
> > 가상 모니터를 사용하고 있습니다. 유즈가 이것을 모니터로 인식하지 못한 모양이다. https://github.com/roshkins/IddSampleDriver
> 
> Yuzu는 OS 자체에서 열거하지 않은 창을 열거하지 않습니다. 설정으로 이동하여 "감지"하고 모니터가 있는지 확인하십시오.

[https://imgdb.in/jhfn](url) Windows detects the virtual monitor.

**Comment by unuruu5760** [comment]
> > > 여전히 문제를 재현할 수 없습니다. 그러나 Windows 설정으로 이동하여 기본 디스플레이를 변경하면 문제가 해결됩니다.
> > 
> > 
> > I am using a virtual monitor. It seems that Yuzu did not recognize this as a monitor. https://github.com/roshkins/IddSampleDriver
> 
> Yuzu will not enumerate windows not enumerated by the OS itself. Make sure to go into settings and "detect" and make sure the monitor is there.

Windows detects the virtual monitor.  https://imgdb.in/jhfn

**Comment by thethiny** [comment]
I ran out of ideas

**Comment by unuruu5760** [comment]
> I ran out of ideas

I solved it using a software called "Borderless Gaming". But I hope that someday yuzu will work without such software.

**Comment by sammypanda** [comment]
yus it would be nice to be able to choose the output display like yuzu --display [index]. Use case: im doing game streaming and it watches the secondary display
